kratos/mpi/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ include_directories( ${CMAKE_SOURCE_DIR}/kratos )
 set( KRATOS_MPI_SOURCES
    mpi_environment.cpp
    sources/mpi_data_communicator.cpp
+   sources/mpi_coloring_utilities.cpp
```

Comment:
There should be an utilities folder in kratos/mpi/ (if not, feel free to create it). I would recommend placing the files for your new utility there.